### PR TITLE
UCP/EP: fix rma config thresholds

### PR DESCRIPTION
**Pull request by evgeny-leksikov**
## What
Fixes RMA short/bcopy/zcopy thresholds

## Why ?
Fixes https://github.com/openucx/ucx/issues/3081
UCX_ZCOPY_THRESH was not applied to ucp_<put|get>*

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8398/ for details (Mellanox internal link).

**Comment by evgeny-leksikov** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8402/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8403/ for details (Mellanox internal link).

File: src/ucp/core/ucp_ep.c [hunk 1]
```diff
@@ -1247,12 +1242,16 @@ void ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config)
...
-            if (iface_attr->cap.flags & UCT_IFACE_FLAG_GET_SHORT) {
-                rma_config->max_get_short = iface_attr->cap.get.max_short;
+            if (iface_attr->cap.flags & UCT_IFACE_FLAG_PUT_BCOPY) {
+                rma_config->max_put_bcopy = ucs_min(iface_attr->cap.put.max_bcopy,
+                                                    rma_config->put_zcopy_thresh);
             }
-            if (iface_attr->cap.flags & UCT_IFACE_FLAG_GET_BCOPY) {
-                rma_config->max_get_bcopy = iface_attr->cap.get.max_bcopy;
+            if (iface_attr->cap.flags & UCT_IFACE_FLAG_PUT_SHORT) {
+                rma_config->max_put_short = ucs_min(iface_attr->cap.put.max_short,
```
[the remark this answers is not in the document]
Comment (evgeny-leksikov) [reply src/ucp/core/ucp_ep.c]: it's initialized above with `SIZE_MAX`, should work as expected.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8405/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8406/ for details (Mellanox internal link).

File: src/ucp/core/ucp_ep.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (evgeny-leksikov) [reply src/ucp/core/ucp_ep.c]: Fixed

**Pull request merged**